src/lang/en.json

```diff
@@ -501,6 +503,8 @@
       },
       "common": {
         "filingPending": "Filing Pending",
+        "paidButNotCompletedByRegistry": "Your submission is still being processed.",
+        "paidButNotCompletedByRegistry1":"Some submissions may take longer than usual to complete. If this issue continues, please contact us.",
```

Comment:
I know these fall under the "paid" code block but I think it's better if the labels match their text.

What do you think about changing these from paid* to pending*? Or make them even closer to their text?